kubernetes-logging/local/README.md

```diff
@@ -0,0 +1,74 @@
+# Containerized-Distributed-Logger 📝
```

Comment:
README should be at the root of the folder, i.e. in ./kubernetes-logging/

Reply:
moved the README to  ./kubernetes-logging/